### PR TITLE
Modified function getCmPerPixel to es6 funcion in src/util/ImageUtil.js

### PR DESCRIPTION
**Pull request by kabuur**
Fixes #598 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by kabuur** [comment]
Modified function getCmPerPixel to es6 funcion in src/util/ImageUtil.js #1164

**Comment by TildaDares** [comment]
Hi @kabuur, the function is fine as it is. You don’t have to change it to an ES6 function. Thank you!

**Comment by TildaDares** [comment]
You are only supposed to change the `var` keyword to `let/const`. Please take a look at the issue description to know the change required https://github.com/publiclab/Leaflet.DistortableImage/issues/1086

File: src/util/ImageUtil.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
 L.ImageUtil = {
 
-  getCmPerPixel(overlay) {
-    var map = overlay._map;
-
-    var dist = map
+  getCmPerPixel:((overlay) =>{
```
Comment (TildaDares) [inline src/util/ImageUtil.js:3]: Hi @kabuur, please revert this change.

File: src/util/ImageUtil.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+    let map = overlay._map;
+    let dist = map
         .latLngToLayerPoint(overlay.getCorner(0))
         .distanceTo(map.latLngToLayerPoint(overlay.getCorner(1)));
 
     return (dist * 100) / overlay.getElement().width;
-  },
+  }),
```
Comment (TildaDares) [inline src/util/ImageUtil.js:10]:
```suggestion
  },
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: src/util/ImageUtil.js (hunk 1 above)
Comment (TildaDares) [reply src/util/ImageUtil.js:3]: Hi @kabuur, this has not been resolved.

File: src/util/ImageUtil.js (hunk 2 above)
Comment (TildaDares) [reply src/util/ImageUtil.js:10]: @kabuur Please revert these changes and leave only the `var` keyword change. Thanks!

**Comment by sashadev-sky** [comment]
Hi! Sorry about this, but we close FTOs with no updates or replies because we assume they have been abandoned. This is to give other contributors a change, as these types of issues are high in demand now. 

If you'd like to try again with a new FTO, we welcome you to!

**Pull request closed without merging**